func main():
  try:
    function();
  catch e:
    println(e);
  end
end
